File: CS/XtraGridTutorial.Step6/ValidationErrorStringsHelper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace XtraGridTutorial
{
    public static class ValidationErrorStringsHelper
    {
        public static string GetDateErrorMessage(string fieldName)
        {
            return string.Format("The {0} field cannot store the future date", fieldName);
        }

        public static string GetAccountErrorMessage(string fieldName)
        {
            return string.Format("You cannot leave the {0} field empty", fieldName);
        }

    }
}

File: VB/XtraGridTutorial.Step5/ValidationErrorStringsHelper.vb
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace XtraGridTutorial
    Public NotInheritable Class ValidationErrorStringsHelper

        Private Sub New()
        End Sub

        Public Shared Function GetDateErrorMessage(ByVal fieldName As String) As String
            Return String.Format("The {0} field cannot store the future date", fieldName)
        End Function

        Public Shared Function GetAccountErrorMessage(ByVal fieldName As String) As String
            Return String.Format("You cannot leave the {0} field empty", fieldName)
        End Function

    End Class
End Namespace
